docs/components/grapesjs.rst

```diff
@@ -0,0 +1,74 @@
+GrapesJS Builder
+################
+
+Currently, Mautic uses the GrapesJS Builder for Emails and Landing pages. 
+
+Setup 
+*****
+
+.. code-block:: BASH
+
+    npm install
+
+.. vale off
+
+Configure Babel, ESLint, Prettier
+=================================
+
+.. vale on
+
+Use the template files provided. For example .eslintrc.temp
+
+How to test standalone
+**********************
+
+.. code-block:: BASH
+
+    npm run start-helloWorld
```

Comment:
As above I think we need to be clear - are we in Mautic's root? In the plugin folder?  What is the difference between the two commands and why might you use one rather than the other?